src/loci/formats/in/ZarrReader.java

```diff
@@ -930,7 +925,9 @@ private void parseOmeroMetadata(String root, Map<String, Object> attr) throws IO
       for (int i = 0; i < channels.size(); i++) {
         Map<String, Object> channel = (Map<String, Object>) channels.get(i);
         Boolean channelActive = (Boolean) channel.get("active");
-        Double channelCoefficient = (Double) channel.get("coefficient");
+        Double channelCoefficient = channel.get("coefficient") instanceof Double ?
```

Comment:
See also #95 an alternative to the `intanceof` check would be to read these as `Number`

Reply:
Ah thanks, so that actually already has been fixed in another PR, should have checked properly.  Maybe we could merge the other PR first...

Reply:
Only obvious caveat is that #95 needs additional work to get the test to compile first. But if that was the preferred option, no objection from my side to splitting these fixes into 2 separate PRs.